test/extended/crc/cmd/cmd.go

```diff
@@ -53,7 +53,7 @@ func (c Command) WithdisableNTP() Command {
 }
 
 func (c Command) ToString() string {
-	cmd := append(c.env(), "crc", c.command)
+	cmd := append(c.env(), "crc", "--log-level", "debug", c.command)
```

Comment:
I would prefer have that log-level as a property on Command, with debug as default if you want, but at least have the change to decide the level when build the command

Reply:
I tried to do this, let me know if this is what you had in mind, or if you were suggesting something else.

Reply:
mostly yes, but even prefer not to limit to debug or default log level.... I mean: 
```
--log-level string   log level (e.g. "debug | info | warn | error") (default "info")
```

so would prefer logLevel: debug on the Command and then just use logLevel property on ToString

Reply:
maybe it will never be used, but in case it is needed somehow we have all the options there